.github/scripts/code_style.sh

```diff
@@ -47,6 +47,8 @@ if [[ "$FIX" == "check" ]]; then
     ./demos/projects/ST/b-l475e-iot01a/port/*.c               \
     ./demos/projects/ST/b-l475e-iot01a/port/*.h               \
     ./demos/projects/ST/b-l475e-iot01a/config/*.h             \
+    ./demos/projects/ST/stm32h745i-disco/cm7/port/*.c               \
```

Comment:
```suggestion
    ./demos/projects/ST/stm32h745i-disco/cm7/port/*.c          \
```
nit: spacing before the `\`s